### PR TITLE
Navigation new

### PR DESCRIPTION
**Pull request by lena-aiu**
@jrmcgarvey Hi John, Now most content from Navbar on left justify. But when I resize the window Navbar content not hiding. I'm working on it. And need help. Thank you

### CODE REVIEW

**Comment by lena-aiu** [comment]
@jrmcgarvey Hi John, please review the changes. Navbar content appears under toaster after resizing the page.

**Comment by lena-aiu** [comment]
@jrmcgarvey HI John, please review changes in Navbar. E helped me with font color for the right side of the navbar and dropdown menu.

**Comment by lena-aiu** [comment]
@jrmcgarvey Please review changes in the Navigation bar. I created a card on Trello board about the tab Other help, sometimes it is not working, and also attached the screenshots with error using development tools. E mentioned that it probably might be resolved on the back-end somehow.

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -8,17 +8,17 @@ class CategoriesController < ApplicationController
...
-      flash.notice = "You are not authorized for that operation."
-      redirect_to home_index_path
-    elsif current_user.role.include?("administrator") || current_user.role.include?("intern")
-      @categories = Category.all
-    else
-      flash.notice = "You are not authorized for that operation."
-      redirect_to home_index_path
-    end
-  end
+  # def index
```
Comment (jrmcgarvey) [inline app/controllers/categories_controller.rb:11]: Why is index commented out?  I think one would want an index view for categories.

**Review by jrmcgarvey — CHANGES_REQUESTED** [review]
There is now a merge conflict that must be resolved.  Also, I don't understand why the index view is commented out for categories -- I think one would want an index view.

File: app/controllers/categories_controller.rb (hunk 1 above)
Comment (lena-aiu) [reply app/controllers/categories_controller.rb:11]: Yes, I forgot to uncomment this, when I was working with the request test. I will fix.

**Comment by lena-aiu** [comment]
@jrmcgarvey Hi John, please review the changes. I forgot to uncomment the index method when was working with the request test. Also, I resolved conflict for Create method in the categories controller. And made changes in categories_request_spec.rb for posting with invalid data.

**Review by jrmcgarvey — APPROVED** [review]
looks ok

**Pull request merged**